<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(C:/Users/yehia_xd1awm4/Downloads/360_F_327515607_Hcps04aaEc7Ki43d1XZPxwcv0ZaIaorh.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; /* Sets the text color to blue */
font-size: 14px; /* Sets the font size to 14 pixels */
font-weight: bold; /* Makes the text bold */
</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(C:/Users/yehia_xd1awm4/Downloads/neumorphic-rectangle-button-free-png.jpeg);
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailText">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px; /* Adjust the value to change the roundness */
background-image: url(C:/Users/yehia_xd1awm4/Downloads/blank-white-background-xbsfzsltjksfompa.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; /* Sets the text color to blue */
font-size: 14px; /* Sets the font size to 14 pixels */
font-weight: bold;

</string>
    </property>
    <property name="text">
     <string>E-Mail Address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordText">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px; /* Adjust the value to change the roundness */
background-image: url(C:/Users/yehia_xd1awm4/Downloads/blank-white-background-xbsfzsltjksfompa.jpg);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; /* Sets the text color to blue */
font-size: 30px; /* Sets the font size to 14 pixels */
font-weight: bold;

</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverValue">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px; /* Adjust the value to change the roundness */
background-image: url(C:/Users/yehia_xd1awm4/Downloads/blank-white-background-xbsfzsltjksfompa.jpg);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portValue">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px; /* Adjust the value to change the roundness */
background-image: url(C:/Users/yehia_xd1awm4/Downloads/blank-white-background-xbsfzsltjksfompa.jpg);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; /* Sets the text color to blue */
font-size: 14px; /* Sets the font size to 14 pixels */
font-weight: bold; /* Makes the text bold */
</string>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black; /* Sets the text color to blue */
font-size: 14px; /* Sets the font size to 14 pixels */
font-weight: bold; /* Makes the text bold */
</string>
    </property>
    <property name="text">
     <string>SMTP-Server:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imapValue">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px; /* Adjust the value to change the roundness */
background-image: url(C:/Users/yehia_xd1awm4/Downloads/blank-white-background-xbsfzsltjksfompa.jpg);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black; /* Sets the text color to blue */
font-size: 14px; /* Sets the font size to 14 pixels */
font-weight: bold; /* Makes the text bold */
</string>
    </property>
    <property name="text">
     <string>IMAP-Server:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
